### PR TITLE
Card section images content model

### PR DESCRIPTION
**Pull request by mfearer94**
I've added the content model for the card section images component. I also added an anchor link and I updated the security of the link to the Box note.

### Changelog

**New**

- Content model
- Anchor link

**Changed**

- Added security to existing Box link

### CODE REVIEW

**Comment by JennySanchez** [comment]
Hi @mfearer94 

For the sentence, "Note: This uses the underlying component "Card group". You can find more information about that component here." 

- We might need to use the Card section – images instead of "This", so the viewer knows what we are talking about here.
- Since this component is "Card section – images", I want to confirm it's "Card group" but not the "Card section".

Thank you 👏.

**Review by photodow — APPROVED** [review]
See some of my comments on other similar components I reviewed.

File: src/pages/components/card-section-images.mdx [hunk 1]
```diff
@@ -24,9 +25,20 @@ Card section &mdash; with images is typically used for presenting resources or l
 
 ![Card section images](../../images/pattern/card-section/card-section-images.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) | Notes | 
+| -------- | --------- | ---------- | ---------- | ----------| ----------|
+| Headline | Short copy | Yes | 1 | Mini (25 / 35)| - |
+| [Card](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/card) | Component | Yes | 2-6 | - | For this component, you must use the image card. |
```
Comment (photodow) [inline src/pages/components/card-section-images.mdx:33]: Ah, wait... I think we're missing the high contrast CTA card.

File: src/pages/components/card-section-images.mdx [hunk 2]
```diff
@@ -24,9 +25,20 @@ Card section &mdash; with images is typically used for presenting resources or l
...
+
```
Comment (photodow) [inline src/pages/components/card-section-images.mdx:34]: Looks like we're missing the inverted CTA here.

**Comment by jeffchew** [comment]
Hi @mfearer94 , would you be able to resolve the merge conflicts? Thanks!

**Pull request merged**